apps/bfd-data-npi/src/test/java/gov/cms/bfd/data/npi/lookup/NPIOrgLookupTest.java

```diff
@@ -66,8 +62,47 @@ public void shouldNotReturnWhenNPINumberIsEmptyAndWhenConstructorSetToTrue() thr
   @Test
   public void shouldNotReturnWhenNPINumberIEmptyStringAndWhenConstructorSetToTrue()
       throws IOException {
-    NPIOrgLookup npiOrgDataLookup = NPIOrgLookup.createNpiOrgLookupForTesting();
-    Optional<String> npiOrgDisplay = npiOrgDataLookup.retrieveNPIOrgDisplay(Optional.of(""));
+    npiOrgDisplay = npiOrgDataLookup.retrieveNPIOrgDisplay(Optional.of(""));
     assertEquals(false, npiOrgDisplay.isPresent());
   }
+
+  /** Should Return Map When Input Stream Is Formatted Correctly. */
+  @Test
+  public void shouldReturnMapWhenInputStreamIsFormattedCorrectly() throws IOException {
+    StringBuilder initialString = new StringBuilder();
+    initialString.append(npiOrgDataLookup.FAKE_NPI_NUMBER);
+    initialString.append("\t");
+    initialString.append(npiOrgDataLookup.FAKE_NPI_ORG_NAME);
+
+    InputStream targetStream = new ByteArrayInputStream(initialString.toString().getBytes());
+    Map<String, String> npiOrgMap = npiOrgDataLookup.readNPIOrgDataStream(targetStream);
+    assertFalse(isNullOrEmptyMap(npiOrgMap));
+    assertEquals(
+        npiOrgDataLookup.FAKE_NPI_ORG_NAME, npiOrgMap.get(npiOrgDataLookup.FAKE_NPI_NUMBER));
+  }
+
+  /** Should Not Return Map When Input Stream Is Not Formatted Correctly. */
```

Comment:
Similarly, what things can constitute formatting incorrectly? When would this occur?

Reply:
Done